configure.in

```
@@ -2254,6 +2254,11 @@ if test -z "$wx_cv_std_libfullpath"; then
     wx_cv_std_libfullpath="/usr/$wx_cv_std_libpath"
 fi
 
+dnl This is Solaris, and needs its pkg-config path adjusted for 64-bit.
+if test "$wx_cv_std_libpath" = "lib/64"; then
+    PKG_CONFIG_PATH="$wx_cv_std_libfullpath/pkgconfig:$PKG_CONFIG_PATH"
```

I don't know if it's really a problem if we get an empty directory in this variable, but perhaps we should test if it's defined before prepending to it, i.e.
```suggestion
    if test -n "$PKG_CONFIG_PATH"; then
      PKG_CONFIG_PATH="$wx_cv_std_libfullpath/pkgconfig:$PKG_CONFIG_PATH"
    else
      PKG_CONFIG_PATH="$wx_cv_std_libfullpath/pkgconfig"
    fi
```

---

I'll do this.